cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -41,3 +64,24 @@ function setDefaultOsImage(props: AwsOrchestratorProps, osImages: AWSVirtualMach
         payload: AwsDefaults.selectDefalutOsImage(osImages),
     } as FormAction);
 }
+
+function setDefaultEC2KeyPair(props: AwsOrchestratorProps, keyPairs: AWSKeyPair[]) {
```

Comment:
nit: if we're only using the `props` object for its `awsDispatch`, how about just sending `awsDispath` as a parameter (instead of the full props object)?